evalml/tests/automl_tests/dask_testing.py

```diff
@@ -0,0 +1,102 @@
+import time
```

Comment:
Style nit-pick: call this file `dask_test_utils.py` instead of `dask_testing.py`